app/src/main/java/com/example/roomremix2/data/FloorPlanData.kt

```diff
@@ -35,4 +39,8 @@ class FloorPlanData {
             Toast.makeText(this, "Error saving floorplan", Toast.LENGTH_SHORT).show()
         }
     }
-}
\ No newline at end of file
+
+    val pathList: List<Path> get() = floorPlan.pathList
+    val colorList: List<Int> get() = floorPlan.colorList
+    val items: List<FloorPlanItem> get() = floorPlan.items
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere? 

Recommendation: if not used, remove it, or keep it in a separate development branch until it is complete.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.